stdeb.cfg

```diff
@@ -1,3 +1,2 @@
 [DEFAULT]
-Recommends: python-pyipputils
 Recommends3: python3-pyipputils
```

Comment:
Does it though? I'm not sure any of that code is present anymore?

Reply:
Good call.